.ci/settings.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash -leE
+
+set -o pipefail
+
+if [ -n "$DEBUG" ]
+then
+    set -x
+fi
+
+module load dev/cuda10.0
+module load hpcx-gcc
+
+SCRIPT_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
+echo "DEBUG: SCRIPT_DIR = ${SCRIPT_DIR}"
+
+echo "DEBUG: CUDA_HOME = ${CUDA_HOME}"
+echo "DEBUG: HPCX_SHARP_DIR = ${HPCX_SHARP_DIR}"
+echo "DEBUG: HPCX_DIR = ${HPCX_DIR}"
+echo "DEBUG: WORKSPACE = ${WORKSPACE}"
+
+HOSTNAME=`hostname -s`
+echo "DEBUG: HOSTNAME = $HOSTNAME"
+
+if [ -z "${WORKSPACE}" ]
+then
+    echo "WARNING: WORKSPACE is empty"
+    WORKSPACE=`cd ${SCRIPT_DIR}/../; pwd -P`
```

Comment:
imo - ```topdir=$(git rev-parse --show-toplevel)``` is a better way o calc scripts_dir. If at some point you will move script to diff location - it will work, while ```cd $var/../``` will not.

Reply:
Agree, fixed.